src/ui/common/src/components/pages/integrations/index.tsx

```diff
@@ -107,10 +148,39 @@ const IntegrationsPage: React.FC<Props> = ({
           <Divider />
         </Box>
 
+        <MetadataStorageInfo serverConfig={serverConfig.config} />
+        {!isLoading && lastFailedFormattedTimestamp && (
+            <Box>
+              <Typography variant="body2" fontWeight="fontWeightRegular" marginTop={2} marginBottom={1}>
+                The last artifact storage migration, which started at {lastFailedFormattedTimestamp}, has failed!
+                As a result, the artifact storage has not changed from `{serverConfig.config?.storageConfig.integration_name}`.
+              </Typography>
+              <Box
+                  sx={{
+                    backgroundColor: theme.palette.red[100],
+                    color: theme.palette.red[600],
+                    p: 2,
+                    paddingBottom: '16px',
+                    paddingTop: '16px',
+                    height: 'fit-content',
+                  }}
+              >
+              <pre style={{ margin: '0px' }}>
+                {`${lastMigration[0].execution_state.error.tip}\n\n${lastMigration[0].execution_state.error.context}`}
+              </pre>
+              </Box>
+            </Box>
+        )}
+
+        <Box marginY={3}>
+          <Divider />
+        </Box>
+
         <Box>
           <Typography variant="h5" marginY={2}>
-            Connected Integrations
+            Connected Data Integrations
```

Comment:
Is this right? Do we have a separate section for connected compute integrations? I thought they were all one section.